internal/drivers/digitalocean/driver.go

```diff
@@ -69,7 +70,7 @@ func (p *config) Create(ctx context.Context, opts *types.InstanceCreateOpts) (in
 		WithField("pool", opts.PoolName).
 		WithField("image", p.image).
 		WithField("hibernate", p.CanHibernate())
-	var name = fmt.Sprintf(opts.RunnerName+"-"+opts.PoolName+"-%d", startTime.Unix())
+	var name = fmt.Sprintf(opts.RunnerName+"-"+opts.PoolName+"-%d", uniuri.NewLen(8)) //nolint
```

Comment:
Nit: will %d work for this as it would be a string?

Reply:
Fixed to %s